src/MakePotCommand.php

```diff
@@ -73,8 +83,9 @@ class MakePotCommand extends WP_CLI_Command {
 	 * @when before_wp_load
 	 */
 	public function __invoke( $args, $assoc_args ) {
-		$this->source = realpath( $args[0] );
-		$this->slug   = Utils\get_flag_value( $assoc_args, 'slug', Utils\basename( $this->source ) );
+		$this->source  = realpath( $args[0] );
+		$this->slug    = Utils\get_flag_value( $assoc_args, 'slug', Utils\basename( $this->source ) );
+		$this->scan_js = ! Utils\get_flag_value( $assoc_args, 'skip-js', ! $this->scan_js );
```

Comment:
I'd prefer the property to be called `$skip_js`, to avoid the weird double negation here. As it is only used once, just directly mapping the parameter seems clearer to me.